### PR TITLE
USC-SIkJalpha Round 0 submission

### PR DESCRIPTION
**Pull request by scc-usc**

### CODE REVIEW

**Comment by Bisaloo** [comment]
Hi @scc-usc :wave:, thanks for your contribution!

There is one last difference with the format used in the US scenario hub you need to take into account. [As detected by our validation checks](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/runs/5551309853?check_suite_focus=true#step:6:19), the `horizon` column should not include the word `ahead`. This is also detailed in our [technical](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/wiki/Targets-and-horizons#date-format) [wiki](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/wiki/Submission-format#horizon-horizon).

This is how the first line of your csv file should change for example:

```diff
- 2022-03-13,A-2022-02-25,inc case,1 wk ahead,2022-03-19,AT,quantile,0.01,20339
+ 2022-03-13,A-2022-02-25,inc case,1 wk,2022-03-19,AT,quantile,0.01,20339
```

Please let me know if this is not clear and you have any questions :relaxed:

**Comment by scc-usc** [comment]
Hi Hugo,

I have made the change. Thanks for catching it.

Ajitesh

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+age_groups: No stratification by age
+importations: Not applicable
+confidence_interval_method: We generate multiple trajectories by (i) Uniform sampling of the recently seen infection rates, under-reporting/asymptomatic population, vaccine coverage, waning parameters uncertainty, and variant prevelance. (ii) For each of the above case trajectories, we use multiple possibilities of death and hospitalization rates that have been seen in the recent past. The above results in multiple points for each day. Quantiles are generated based on sampling among these points for each day.
+calibration: All calibrations using regression. No manual-tuning other than the listed assumption and trivial settings: non-zero lag between infection and death.
+spatial_structure: Not applicable.
+team_funding: National Science Foundation Award # 2135784
+data_inputs: Cases and deaths - JHU
+citation: 
+	[1] Ajitesh Srivastava, Tianjian Xu and Viktor K. Prasanna, "Fast and Accurate Forecasting of COVID-19 Deaths using the SIkJalpha Model". https://arxiv.org/abs/2007.05180.
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:30]:
```suggestion
citation: |
  [1] Ajitesh Srivastava, Tianjian Xu and Viktor K. Prasanna, "Fast and Accurate Forecasting of COVID-19 Deaths using the SIkJalpha Model". https://arxiv.org/abs/2007.05180.
```

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+methods_long: [Note that this is general description. Hospitalizaitons and age-groups are not applicable to European Scenario Modeling Hub] We use our own epidemic model called SI-kJalpha, preliminary version
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:31]:
```suggestion
methods_long: |
  [Note that this is general description. Hospitalizaitons and age-groups are not applicable to European Scenario Modeling Hub] We use our own epidemic model called SI-kJalpha, preliminary version
```

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+natural_immunity_duration: Same as above
+case_fatality_rate: Learned from recent data.
+infection_fatality_rate: Learned from recent data.
+asymptomatics: Similar to the range obtained for the US states using the CDC sero-prevelance data from the CDC. Note that this covers the currect cumulative fraction of unreported, untested, and asymptomatic.
+age_groups: No stratification by age
+importations: Not applicable
+confidence_interval_method: We generate multiple trajectories by (i) Uniform sampling of the recently seen infection rates, under-reporting/asymptomatic population, vaccine coverage, waning parameters uncertainty, and variant prevelance. (ii) For each of the above case trajectories, we use multiple possibilities of death and hospitalization rates that have been seen in the recent past. The above results in multiple points for each day. Quantiles are generated based on sampling among these points for each day.
+calibration: All calibrations using regression. No manual-tuning other than the listed assumption and trivial settings: non-zero lag between infection and death.
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:25]:
```suggestion
calibration: 'All calibrations using regression. No manual-tuning other than the listed assumption and trivial settings: non-zero lag between infection and death.'
```

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+team_model_designation: primary
+methods: Uses SIKJalpha which models temporally varying infection, death, and hospitalization rates. Learning is performed by reducing the problem to multiple simple linear regression problems. Effects of variants are modeled separately as competing contagions.
+modeling_NPI: No NPI included, contact changes over time derived from Cuebiq data.
+compliance_NPI: Not modeled.
+contract_tracing: Not applicable.
+testing: Not applicable.
+vaccine_efficacy_transmission: Not explicitly modeled
+vaccine_efficacy_delay: 14 days
+vaccine_hesitancy: A "suceptible-infected"model. Boosters are modeled such that the "susceptible" population is the eligible population who are yet to get a booster. To achieve the desired saturation level to create sub-scenarios, we increase the adoption rate over time.
+vaccine_immunity_duration: Only waning immunity is considered! The probability of transferring to partially immune state by at time t is modeled as a gamma distribution such that: (i) the median is as per the scenario; (ii) the efficacy (given the partial immune protection against infection) after the first 60 days is the "initial" vaccine efficacy.
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:17]:
```suggestion
vaccine_immunity_duration: 'Only waning immunity is considered! The probability of transferring to partially immune state by at time t is modeled as a gamma distribution such that: (i) the median is as per the scenario; (ii) the efficacy (given the partial immune protection against infection) after the first 60 days is the "initial" vaccine efficacy.'
```

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 5]
```diff
@@ -0,0 +1,54 @@
+team_name: University of Southern California
+model_name: SI_kJalpha
+model_abbr: USC-SI_kJalpha
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:3]:
```suggestion
model_abbr: USC-SIkJalpha
```

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+model_version: 2022-03-13
+model_contributors: Ajitesh Srivastava <ajiteshs@usc.edu>
+website_url: https://scc-usc.github.io/ReCOVER-COVID-19
+license: mit
+team_model_designation: primary
+methods: Uses SIKJalpha which models temporally varying infection, death, and hospitalization rates. Learning is performed by reducing the problem to multiple simple linear regression problems. Effects of variants are modeled separately as competing contagions.
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:9]: Methods should be under 200 characters. Feel free to expand in the `methods_long` section.

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+modeling_NPI: No NPI included, contact changes over time derived from Cuebiq data.
+compliance_NPI: Not modeled.
+contract_tracing: Not applicable.
```
Comment (Bisaloo) [inline data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:12]:
```suggestion
contact_tracing: Not applicable.
```

**Review by Bisaloo — CHANGES_REQUESTED** [review]
Hi again, I've added a couple of comments to fix the yaml file structure and make sure it passes our validation.
There is a final issue with the `model_version` field that is currently not defined in our template. We are discussing if it's something we should add and I'll get back to you shortly with more information!

**Comment by Bisaloo** [comment]
Hi @scc-usc, I have updated our validations to allow the `model_version` field. Would you have a moment to check at the commit suggestions I made to fix the metadata file format?

If you're busy and do not want to deal with it, I can do the changes myself (none of the actual content will change, it's just formatting).

File: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt (hunk 6 above)
Comment (scc-usc) [reply data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt:9]: Hi Hugo. Sorry for the late response. I was traveling and now catching up with deadlines. I have accepted your suggestions. For this one, could you please remove the last sentence? 
Thanks,
Ajitesh

**Review by Bisaloo — APPROVED** [review]

**Comment by scc-usc** [comment]
@Bisaloo : I just noticed that the validation had failed. But I also see "approved these changes". Do I need to resubmit or the submission has already been fixed?
Thanks,
Ajitesh

**Comment by Bisaloo** [comment]
:wave: @scc-usc, yes, everything was fine when I approved the PR. 

However, the most recent changes have some issues though. From line 306177 onwards, the horizon column uses `1 wk ahead` instead of `1 wk`, as discussed in https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/pull/4#issuecomment-1073779872. Would you like to fix it?

We are waiting for a couple more submissions are will probably merge them all together at this moment :slightly_smiling_face:

**Pull request merged**